default.nix

```diff
@@ -40,7 +40,7 @@ rec {
 
   plutus-chain-index = plutus-apps.haskell.packages.plutus-chain-index.components.exes.plutus-chain-index;
 
-  marconi = plutus-apps.haskell.packages.marconi.components.exes.marconi;
+  marconi = plutus-apps.haskell.packages.marconi-core.components.exes.marconi-chain-index;
```

Comment:
That seems wrong. I think it should be `marconi-chain-index`, not `marconi-core`